internal/bminventory/inventory.go

```diff
@@ -3960,3 +4084,16 @@ func (b *bareMetalInventory) setPullSecretFromOCP(cluster *common.Cluster, log l
 	setPullSecret(cluster, pullSecret)
 	return nil
 }
+
+func (b *bareMetalInventory) configureStaticIPs(cluster *common.Cluster, staticIpsConfig []*models.StaticIPConfig) string {
+	lines := make([]string, len(staticIpsConfig))
+
+	// construct static IPs config string
+	for i, entry := range staticIpsConfig {
+		lines[i] = fmt.Sprintf("%s;%s;%s;%s;%s", entry.Mac, entry.IP, entry.Mask, entry.DNS, entry.Gateway)
+	}
+
+	sort.Strings(lines)
```

Comment:
Why do you need sorting?

Reply:
user can enter configuration for different macs in whatever order he wants. i am sorting in order to check if the total configuration ( for all the macs) has changed or not

Reply:
Good point.